web/src/search_suggestion.ts

```diff
@@ -675,15 +675,21 @@ function get_sent_by_me_suggestions(last: NarrowTerm, terms: NarrowTerm[]): Sugg
     return [];
 }
 
-function get_operator_suggestions(last: NarrowTerm): Suggestion[] {
+function get_is_operator_suggestions(last: NarrowTerm): Suggestion[] {
+    if (!(last.operator === "is")) {
+        return [];
+    }
```

Comment:
The logic is okay to move the `is:private` check out of the `get_operator_suggestions` but we already have a `get_is_filter_suggestions`. I think handling it in there would be more appropriate rather than having another `get_is_operator_suggestions` only for `is:private`

Reply:
ah ok, makes sense to me. I think we can just remove the logic since the existing implementation of `get_is_filter_suggestions` will already suggest `is:dm` for `is:` and `is:private`